### PR TITLE
Team/sultans of swing/fix/door interaction

### PR DESCRIPTION
**Pull request by smiljanaspasic**

### CODE REVIEW

File: source/dagger/gameplay/team_game/door_interaction.cpp [hunk 1]
```diff
@@ -15,52 +15,58 @@ using namespace team_game;
...
+	auto viewkey = Engine::Registry().view<Key>();
 	for (auto keyentity : viewkey)
 	{
 		auto& key = viewkey.get<Key>(keyentity);
 		if (key.GetPickedUp() == true) {
-			keytaken = true;
+			pickedup = true;
 			Engine::Registry().destroy(keyentity);
 		}
+		else { pickedup = false; }
```
Comment (DPrivalov) [inline source/dagger/gameplay/team_game/door_interaction.cpp]: try to follow some codestyle:
put "{" after new line or not, decide this and then do it in one way. 

I see different ways inside one file

File: source/dagger/gameplay/team_game/door_interaction.cpp [hunk 2]
```diff
@@ -15,52 +15,58 @@ using namespace team_game;
 
 using namespace dagger;
 
-bool keytaken = false;
 
+Bool pickedup = false;
```
Comment (DPrivalov) [inline source/dagger/gameplay/team_game/door_interaction.cpp]: can't u put it inside DoorSystem at least? as a member field

File: source/dagger/gameplay/team_game/team_game_main.cpp [hunk 3]
```diff
@@ -116,78 +116,83 @@ void SetupWorldJovica(Engine& engine_, Registry& reg_)
 }
 void SetupWorldSmiljana(Engine& engine_, Registry& reg_) {
 
-/*
+    /*
```
Comment (DPrivalov) [inline source/dagger/gameplay/team_game/team_game_main.cpp]: too many white spaces/new lines, clean it up a bit before merging.

**Review by DPrivalov — APPROVED** [review]

File: source/dagger/gameplay/team_game/door_interaction.cpp (hunk 2 above)
Comment (smiljanaspasic) [reply source/dagger/gameplay/team_game/door_interaction.cpp]: Thank you for all of the nice suggestions 😃

**Review by Yugo0 — APPROVED** [review]

**Pull request merged**